src/contingency-flow.js

```diff
@@ -49,6 +50,11 @@ let ContingencyComponent : Component<ContingencyProps> = create({
     function close(event) : ZalgoPromise<void> {
       event.preventDefault();
       event.stopPropagation();
+
+      if (contingencyResolveFunction) {
+        contingencyResolveFunction({ success: false });
```

Comment:
Is there a better way to resolve the `start` method with `success: false` then keeping a reference to the `resolve` function?